src/annotator/config/index.js

```diff
@@ -18,6 +18,14 @@ function configFrom(window_) {
     // URL of the client's boot script. Used when injecting the client into
     // child iframes.
     clientUrl: settings.clientUrl,
+    disableToolbarCloseBtn:settings.hostPageSetting('disableToolbarCloseBtn', {defaultValue: true}),
+    disableToolbarMinimizeBtn: settings.hostPageSetting('disableToolbarMinimizeBtn'),
+    disableToolbarHighlightsBtn: settings.hostPageSetting('disableToolbarHighlightsBtn'),
+    disableToolbarNewNoteBtn: settings.hostPageSetting('disableToolbarNewNoteBtn'),
+    disableBucketBar: settings.hostPageSetting('disableBucketBar'),
+    enableAnnotatorFrameDropShadow: settings.hostPageSetting('enableAnnotatorFrameDropShadow'),
```

Comment:
Is there a reason that the drop shadow should be configurable separately from the theme or is this inherently a part of the "clean" theme?

If we do decide to keep this setting I would name it something like `enableSidebarDropShadow` because the "annotator" name in the CSS code dates back to Annotator JS.